Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -285,6 +285,7 @@ class DefaultBrowserOnboardingViewController: UIViewController {
     
     @objc private func goToSettings() {
         viewModel.goToSettings?()
+        UserDefaults.standard.set(true, forKey: "DidDismissDefaultBrowserCard") // Don't show default browser card if this button is clicked
```

Comment:
This won't dismiss the default browser card right away but will do so when user reopens the app or if bvc gets reloaded.